tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -1334,6 +1379,73 @@ void dispatchTest(ID3D12Device *D3DDevice, bool VerboseLogging,
   }
 }
 
+static bool isWarpDevice(ID3D12Device *D3DDevice) {
```

Comment:
I'm very surprised to see this here - especially since we created the device, we should know if it is WARP or not, seems weird we need to go and find this out again.

Reply:
Ah, I can remove this and store as a member now. Originally, I wanted this in a non-member function.